Content.Shared/_DV/CosmicCult/Components/CosmicAstralNovaComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Shared.Damage;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._DV.CosmicCult.Components;
+
+/// <summary>
+/// Component to call back to the cosmic cult ability system regarding a collision
+/// </summary>
+[NetworkedComponent, RegisterComponent]
+public sealed partial class CosmicAstralNovaComponent : Component
+{
+    [DataField]
+    public bool DoStun = true;
+    public DamageSpecifier CosmicNovaDamage = new()
+    {
+        DamageDict = new() {
+            { "Asphyxiation", 13 }
+        }
+    };
```

Comment:
This is not a datafield, it is unused, and it's missing a blank line between fields